tests/10apidoc/33room-members.pl

```diff
@@ -61,12 +61,17 @@ sub matrix_join_room
    is_User( $user ) or croak "Expected a User; got $user";
 
    my %content;
+   my %params;
+   if ( defined $opts{server_name} ) {
```

Comment:
FYI, you could write this the same way as the following line for consistency:

```perl
defined $opts{server_name} and $params{server_name} = $opts{server_name};
```

Another common idiom is something like:

```perl
   foreach (qw( server_name )) {
      $params{$_} = $opts{$_} if exists $opts{$_};
   }
```

-- which obviously scales better for multiple options.

Reply:
> FYI, you could write this the same way as the following line for consistency:
> 
> ```perl
> defined $opts{server_name} and $params{server_name} = $opts{server_name};
> ```

ahh, quite bash-like I see.